test/data/IncrementalMerkleTree.ts

```diff
@@ -121,6 +121,24 @@ describe('IncrementalMerkleTree', function () {
 
       expect(await instance.callStatic.root()).to.equal(hash);
     });
+
+    it('POC: returns Merkle root derived from elements contained in unbalanced tree', async () => {
```

Comment:
What is "POC"?

Reply:
This test seems a bit redundant.  Though, there's a lot of overlap in several of the tests.